index.html

```diff
@@ -69,12 +72,236 @@ <h1 class="col col-11 col-lg-5 aligne-self-center"> glade to see you</h1>
+                </ul>
+                <a href="#" class="button">Get my resume</a>
+
+                <div class="accordion" id="accordionExample">
+                   
+                      <div class="d-header" id="headingOne">
+                        <h5 class="mb-0">
+                          <button class="btn" type="button" data-toggle="collapse" data-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
+                            Languages
+
+                          </button>
+                        </h5>
+                      </div>
+                  
+                      <div id="collapseOne" class="collapse show" aria-labelledby="headingOne" data-parent="#accordionExample">
+                        <div class="card-body">
+                          <div class="language">
+                              <img class="language-img" src="./assets/js.png" alt="js">
+                              <span>JavaScript</span>
+                          </div>
+                          <div class="language">
+                            <img class="language-img" src="./assets/html.png" alt="js">
+                            <span>HTML</span>
+                        </div>
+                        <div class="language">
+                            <img class="language-img" src="./assets/css.png" alt="js">
+                            <span>CSS</span>
+                        </div>
+                        </div>
+                      </div>
+                    
+                    
+                      <div class="d-header" id="headingTwo">
+                        <h5 class="mb-0">
+                          <button class="btn collapsed" type="button" data-toggle="collapse" data-target="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
+                            Frameworks
+                          </button>
+                        </h5>
+                      </div>
+                      <div id="collapseTwo" class="collapse" aria-labelledby="headingTwo" data-parent="#accordionExample">
+                        <div class="card-body">
+                          
+                        </div>
+                      </div>
+                    
+                    
+                      <div class="d-header" id="headingThree">
+                        <h5 class="mb-0">
+                          <button class="btn collapsed" type="button" data-toggle="collapse" data-target="#collapseThree" aria-expanded="false" aria-controls="collapseThree">
+                            skills
+                          </button>
+                        </h5>
+                      </div>
+                      <div id="collapseThree" class="collapse" aria-labelledby="headingThree" data-parent="#accordionExample">
+                        <div class="card-body">
+                         
+                        </div>
+                      </div>
+                    
+                  </div>
```

Comment:
- [ ] The technology categories ("Languages", "Frameworks" and "Skills") is a list and should use the `ul` for grouping, while each of them should have a `li` tag
- [ ] The technology in each category (e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`
- [ ] The technology in each category (e.g., "JavasScript", "HTML" and "CSS") should also be embedded inside the `Languages`

Reply:
- This was not addressed well let me help you to understand what it means
you can do something like 
<ul>
  <li> Languages
    <ul>
       <li>Javascript</li>
       <li>...</li>
     </ul>
    </li>
</ul>
So that Languages, framework, skills they will be a list that have list inside each and every one